Automattic/jetpack
# WP Super Cache: preload notification shows last 5 preloaded files now

When the plugin is preloading pages it displays the last preloaded URL. This patch changes that so it show the last 5 preloaded URLs, with the newest URL coming up at the bottom, and the oldest being removed off the top.

### Other information:


## Jetpack product discussion


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Apply patch
Run a preload and  ten seconds or so after it starts,  you will see the last 5 URLs that have been preloaded appear in the admin notice area of the preload page.

![BEFORE](https://github.com/Automattic/jetpack/assets/5656673/f001ea6e-060c-4633-b61a-057f2a7fac7e)
Before


![AFTER](https://github.com/Automattic/jetpack/assets/5656673/a99de418-419a-4195-b904-176c326dfb98)
After
